libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -89,11 +89,14 @@ string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
 			return Whiskers(R"(
 				function <functionName>(src, dst, length) {
 					calldatacopy(dst, src, length)
+					<?cleanup>
 					// clear end
 					mstore(add(dst, length), 0)
+					</cleanup>
```

Comment:
> else only the exact length will be written

Isn't this "else" part missing here?

Reply:
What else would you want to happen in the `else` part, other than not cleaning the end?

Reply:
Ah, wait, I think I read too much into your comment. I think it's correct and I was just thinking about the thing I suggested on the chat - i.e. that we should clean the exact number of bytes instead of not cleaning at all.

But yeah, I need to look at the usage to see if we actually need this. Maybe no cleanup is what we need after all...